docs/bgp-policy.md

```diff
@@ -47,9 +49,10 @@ data:
 
 A BGPPolicy in Kubernetes is a Custom Resource Definition (CRD) object.
 
-The following manifest creates a BGPPolicy object. It will start a BGP process with ASN `64512`, listening on port `179`,
-on Nodes labeled with `bgp=enabled`. The process will advertise LoadBalancerIPs and ExternalIPs to a BGP peer at IP
-address `192.168.77.200`, which has ASN `65001` and listens on port `179`:
+The following manifest creates a BGPPolicy object. It will start a BGP process with the private ASN `64512`, listening on
+port `179`, on Nodes labeled with `bgp=enabled`. The process will use the confederation identifier `65000` and advertise
+LoadBalancerIPs and ExternalIPs. It peers with an external BGP peer at IP address `192.168.77.200`, which has ASN `65001`
+and listens on port `179`, as well as internal BGP peers within the confederation.
```

Comment:
you already have a dedicated confederation section / example below, so I would not modify this example which should demonstrate basic usage

Reply:
Make sense.